api/desecapi/metrics.py

```diff
@@ -0,0 +1,28 @@
+from prometheus_client import Counter
+
+
+metrics = {}
+
+
+def get(name):
+    return metrics.get(name)
+
+
+def set(name, *args, **kwargs):
```

Comment:
It turns out that `set` is a bad function name (there is a Python function with the same name that gets overridden when doing that). Let's change it to `set_counter`. My bad!